clang-tools-extra/test/clang-tidy/checkers/portability/template-virtual-member-function.cpp

```diff
@@ -171,3 +171,20 @@ struct NoInstantiation<int, U>{
     };
 };
 } // namespace PartialSpecializationNoInstantiation
+
+namespace PR139031 {
```

Comment:
nit: IMO `PR-XXXX` or `GH-XXX` namespaces just bloat code without any benefits, `git blame` can do the job.
And if LLVM ever moves from GitHub issue, this number will become meaningless.

```suggestion
namespace PureVirtual {
```